Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -3655,6 +3660,21 @@ namespace AssetProcessor
                     continue;
                 }
             }
+            else if(!AZ::IO::PathView(databaseSourceName.toUtf8().constData()).IsAbsolute())
+            {
+                QString absolutePath = m_platformConfig->FindFirstMatchingFile(databaseSourceName);
+
+                if (!absolutePath.isEmpty())
+                {
+                    databaseSourceName = absolutePath;
+                }
+                else
+                {
+                    // If we can't resolve the dependency, it usually means it doesn't exist
+                    job.m_jobDependencyList.erase(jobDependencyInternal);
```

Comment:
so the tech does not support "future dependent" files that might be created from another job, correct?

Reply:
This is for job dependencies specifically, so that is correct.  However this is also just part of the in-memory dependency resolution, the dependency should still be stored in the db beforehand.